src/main/typescript/ruleEngine/ruleElement.ts

```diff
@@ -218,7 +218,7 @@ export default class RuleElement {
     }
 
     if (this.rangesRegexpMatch && document.data.type === TYPES.ITEM.WEAPON) {
-      return document.data.data.ranges
```

Comment:
This doesn't seem intentional, please check.

Reply:
Huh. No idea how that happened.